### PR TITLE
Update reattach-pv README

### PR DESCRIPTION
**Pull request by thbkrkr**
Follow-up of #6118.

### CODE REVIEW

File: support/reattach-pv/README.md [hunk 1]
```diff
@@ -30,49 +30,19 @@ Flags:
...
 # build the binary with a recent Go version
 go build
 
 # re-create the cluster with the same name
-./reattach-pv --elasticsearch-manifest elasticsearch.yml --dry-run
+./reattach-pv --elasticsearch-manifest cluster-A.yml --dry-run
 
-# optionally re-create the cluster with a new name
+# or re-create the cluster with a new name
 reattach-pv --elasticsearch-manifest cluster-B.yml --old-elasticsearch-name cluster-A --dry-run
```
Comment (naemono) [inline support/reattach-pv/README.md:43]: Unfortunately, this typo was my own doing.
```suggestion
./reattach-pv --elasticsearch-manifest cluster-B.yml --old-elasticsearch-name cluster-A --dry-run
```

**Review by naemono — APPROVED** [review]
LGTM.  Thanks for the follow-up @thbkrkr

**Comment by thbkrkr** [comment]
buildkite test this

**Comment by thbkrkr** [comment]
buildkite test this

**Pull request merged**